hydroflow/tests/surface_codegen.rs

```diff
@@ -23,13 +20,13 @@ use tokio::task::LocalSet;
 // TODO(mingwei): Documentation articles.
 // TODO(mingwei): Find a way to display join keys
 
-fn recv_into_vec<T>(recv: &mut UnboundedReceiver<T>) -> Vec<T> {
-    std::iter::from_fn(|| recv.try_recv().ok()).collect()
+fn recv_into_vec<T>(recv: &mut UnboundedReceiverStream<T>) -> Vec<T> {
+    std::iter::from_fn(|| recv.as_mut().try_recv().ok()).collect()
```

Comment:
This is kind of a syntactic bummer. Would we always need to say `.as_mut`? If so can we hide this? If not, what is the common case and can we have longer syntax only for the uncommon case?

Reply:
Common case shouldn't have the user interacting directly with `UnboundedReceiver[Stream]` much at all